configure.ac

```diff
@@ -323,7 +323,7 @@ fi
 
 
 dnl detect a usable system regular expression library
-AC_CHECK_LIB(c, regcomp, LIBREGEX="-lc",
+AC_CHECK_LIB(root, regcomp, LIBREGEX="-lroot",
```

Comment:
I can't tell in context if this is a BeOS-specific portion or not, but I'm not sure it's a good idea to remove `-lc`; extending to check `-lroot` is fine.

Reply:
Fixed.